spec/v2/index.bs

```diff
@@ -933,6 +943,17 @@ The following properties are defined for data type <{Assurance}>:
   <figcaption>Properties of data type Assurance</figcaption>
 </figure>
 
+<div class=example>
+  Example value for the case that 42% of the product's overall GHG emissions covered by the data quality assessment:
+  <pre highlight=json>
+  {
+    "providerName": "",
+    "assurance": true,
+    "coveragePercent": 42.0,
+  }
```

Comment:
I think this example is still not correct, it seems to be a strange mix between Data Type: Assurance and Data Type: DataQualityIndicators 

Let's make it a correct example for Data Type: Assurance. 

- It must only include properties of this Data Type: `coveragePercent` is NOT an attribute of Data Type: Assurance, but rather of Data Type: DataQualityIndicators
- It must follow the specification of Data Type: Assurance (providerName must be non-empty, and coverage must be populated with a value listed in the spec.

I am meanwhile working with Joy on more realistic "fake data", so you could already use the following:
Example value for the case that the PCF was assured by a third-party:
{
    "assurance": true,
    "coverage": "PCF system",
    "level": "limited"
    "boundary": "Cradle-to-Gate
    "providerName": "My Auditor",
    "completedAt": "2022-12-08T14:47:32Z"
    "standardName": "ISO 14044"
  }

Reply:
My bad! I thought that the only field missing was `providerName` (and that it could be an empty string) and did not even notice that the other one was wrong. Thank you so much for the example! I have used it in the lates commit.